docs/installation.md

```diff
@@ -17,14 +17,21 @@ programs:
 The most recent version of VirtualBox, [VirtualBox 5.0](https://www.virtualbox.org/wiki/Changelog), is [not compatible](https://github.com/gamechanger/dusty/issues/383) with the current version of boot2docker.
 Please install <strong>VirtualBox 4.X</strong> in order to run Dusty.
 
+## Pre-Dusty Install
+
 ```
 brew update
 which nginx || brew install nginx
 which boot2docker || brew install boot2docker
 which docker-compose || brew install docker-compose
 ```
+```
+ssh-add -K
+  -or-
+ssh-add -K <path-of-private-key>
```

Comment:
Should say a little bit here about why we need this / which private key we need.